utils/Recorder/Recorder.ts

```diff
@@ -60,14 +61,27 @@ export async function defaultRecorder(hre: HardhatRuntimeEnvironment, params: De
 export async function defaultTimelockRecorder(hre: HardhatRuntimeEnvironment, params: DefaultRecorderParams | null = null) {
   const blockBefore = await hre.ethers.provider.getBlock("latest");
   const timestamp = blockBefore.timestamp;
-  const days = params?.executionStartInDays ?? 14;
 
-  const secondsInDay = 60 * 60 * 24;
-  const eta = timestamp + days * secondsInDay + 100;
+  // the default use of the timlock recorder is to queue the transactions
+  // be default they will be executed by the multisig
+  // if we want to execute the approved transactions batch from terminal we should specify the signer explicitly (the executor)
+  const signer = params?.singer ?? (await getProposer(hre));
+
+  const timelock = (await getTimelock(hre)).connect(signer);
```

Comment:
Would that work also for chains we did not deploy the timelock? Or our assumption is that we'll always deploy the timelock?

I'm thinking if we'll need it on testnets or not.. My assumption is no because we don't want to wait too long for that (and also because we don't have a multisig on most testnets - like arbitrum testnet for example..)